code/modules/reagents/Chemistry-Reagents/Consumable-Reagents/Food-Reagents.dm

```diff
@@ -410,19 +410,412 @@ datum/reagent/consumable/eggyolk
+	reagent_state = LIQUID
 	color = "#C8A5DC"
 
-datum/reagent/consumable/corn_syrup
+/datum/chemical_reaction/corn_syrup
+	name = "corn_syrup"
+	id = "corn_syrup"
+	result = "corn_syrup"
+	required_reagents = list("corn_starch" = 1, "sacid" = 1)
+	result_amount = 2
+	required_temp = 374
+	mix_message = "The mixture forms a viscous, clear fluid!"
+
+datum/reagent/corn_syrup
 	name = "Corn Syrup"
 	id = "corn_syrup"
-	description = "Decays into sugar."
+	description = "A sweet syrup derived from corn starch that has had its starches converted into maltose and other sugars."
+	reagent_state = LIQUID
 	color = "#C8A5DC"
 
-datum/reagent/consumable/corn_syrup/on_mob_life(var/mob/living/M as mob)
-	M.reagents.add_reagent("sugar", 3)
-	M.reagents.remove_reagent("corn_syrup", 1)
+datum/reagent/corn_syrup/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	M.reagents.add_reagent("sugar", 1.2)
+	..()
+	return
+
+/datum/chemical_reaction/vhfcs
+	name = "vhfcs"
+	id = "vhfcs"
+	result = "vhfcs"
+	required_reagents = list("corn_syrup" = 1)
+	required_catalysts = list("enzyme" = 1)
+	result_amount = 1
+	mix_message = "The mixture emits a sickly-sweet smell."
+
+datum/reagent/vhfcs
+	name = "Very-high-fructose corn syrup"
+	id = "vhfcs"
+	description = "An incredibly sweet syrup, created from corn syrup treated with enzymes to convert its sugars into fructose."
+	reagent_state = LIQUID
+	color = "#C8A5DC"
+
+datum/reagent/vhfcs/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	M.reagents.add_reagent("sugar", 2.4)
+	..()
+	return
+
+/datum/chemical_reaction/cola
+	name = "cola"
+	id = "cola"
+	result = "cola"
+	required_reagents = list("carbon" = 1, "oxygen" = 1, "water" = 1, "sugar" = 1)
+	result_amount = 4
+	mix_message = "The mixture begins to fizz."
+
+/datum/reagent/honey
+	name = "Honey"
+	id = "honey"
+	description = "A sweet substance produced by bees through partial digestion. Bee barf."
+	reagent_state = LIQUID
+	color = "#CFCF1F"
+
+datum/reagent/honey/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	M.reagents.add_reagent("sugar", 0.8)
+	..()
+	return
+
+
+/datum/reagent/chocolate
+	name = "Chocolate"
+	id = "chocolate"
+	description = "Chocolate is a delightful product derived from the seeds of the theobroma cacao tree."
+	reagent_state = LIQUID
+	color = "#2E2418"
+
+datum/reagent/chocolate/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	M.reagents.add_reagent("sugar", 0.8)
+	..()
+	return
+
+/datum/reagent/porktonium
+	name = "Porktonium"
+	id = "porktonium"
+	description = "A highly-radioactive pork byproduct first discovered in hotdogs."
+	reagent_state = LIQUID
+	color = "#AB5D5D"
+	metabolization_rate = 0.2
+	overdose_threshold = 125
+
+datum/reagent/porktonium/overdose_process(var/mob/living/M as mob)
+	if(volume > 125)
+		if(prob(8))
+			M.reagents.add_reagent("cyanide", 10)
+			M.reagents.add_reagent("radium", 15)
```

Comment:
That's right, people. Eat too many hot dogs/pot noodles and you too will fill with cyanide and radium.